### ISSUE TITLE
Formatting breaks json-ld script blocks 

### ISSUE THREAD

**Issue by DPangerl** [panoply/vscode-liquid#79]
I'm working with the shopify default dawn theme and there are some microdata blocks in the header.liquid. 
If i try to format the document, everything seems to work fine, but the json-ld blocks are broken afterwards. 

It goes from:

```

<script type="application/ld+json">
    {
      "@context": "http://schema.org",
      "@type": "Organization",
      "name": {{ shop.name | json }},
      {% if section.settings.logo %}
        {% assign image_size = section.settings.logo.width | append: 'x' %}
        "logo": {{ section.settings.logo | img_url: image_size | prepend: "https:" | json }},
      {% endif %}
      "sameAs": [
        {{ settings.social_twitter_link | json }},
        {{ settings.social_facebook_link | json }},
        {{ settings.social_pinterest_link | json }},
        {{ settings.social_instagram_link | json }},
        {{ settings.social_tiktok_link | json }},
        {{ settings.social_tumblr_link | json }},
        {{ settings.social_snapchat_link | json }},
        {{ settings.social_youtube_link | json }},
        {{ settings.social_vimeo_link | json }}
      ],
      "url": {{ shop.url | append: page.url | json }}
    }
  </script>

```

to

```

<script type="application/ld+json">

[36mLine: 1[0m
[31m[36 mLine : 1 [[0m
[31m        0 m [[0m
[31m            31 m {[0m
[31m                [[0m
[31m                    0 m [[0m
[31m                        31 m "@context" : "http://schema.org",[0m
[31m                        [[0m
</script>

```

I've copied this block in a blank liquid file… same problem. 

Is there any way, to exclude formatting those blocks? 
Or can i adjust some other setting? 

If you need further information, just ask. 
Thanks for the otherwise great extension 👍

**Comment by panoply** [comment]
#38 is a temporary workaround for this.

**Comment by panoply** [comment]
🚢  Shipped [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)
